src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -33,16 +33,28 @@ public Bootstrapper(ITemplateEngineHost host, Action<IEngineEnvironmentSettings>
             _templateCreator = new TemplateCreator(_engineEnvironmentSettings);
         }
 
+        [Obsolete("Use ITemplateEngineHost.BuiltInComponents or AddComponent to add components.")]
         public void Register(Type type)
         {
             _engineEnvironmentSettings.SettingsLoader.Components.Register(type);
         }
 
+        [Obsolete("Use ITemplateEngineHost.BuiltInComponents or AddComponent to add components.")]
         public void Register(Assembly assembly)
         {
             _engineEnvironmentSettings.SettingsLoader.Components.RegisterMany(assembly.GetTypes());
         }
 
+        /// <summary>
+        /// Adds component to manager, which can be looked up later via <see cref="IComponentManager.TryGetComponent{T}(Guid, out T)"/> or <see cref="IComponentManager.OfType{T}"/>.
```

Comment:
I think it's worth adding that the components are not saved anymore, so they needed to be added each time instance is created.